tensorflow/python/keras/layers/advanced_activations.py

```diff
@@ -330,3 +330,41 @@ def get_config(self):
   @tf_utils.shape_type_conversion
   def compute_output_shape(self, input_shape):
     return input_shape
+
+
+@keras_export('keras.layers.NoisyReLU')
+class NoisyReLU(Layer):
+  """Noisy Rectified Linear Unit activation function.
+
+  It follows:
+  `f(x) = x+Y for x > 0, where Y is noise`,
```

Comment:
Isn't Y a Gaussian noise? why is it a constant here?

Reply:
I just updated the comment, but I am using noise from the user end.

Reply:
I mean isn't Y a Gaussian noise such that every time ._call_() happens it should be a different value?

Reply:
The noise value is between 0 to standard deviation of the inputs given. It is upon the user how much noise is to be added to the actual relu function.